ui-modules/app-inspector/app/components/providers/entity-api.provider.js

```diff
@@ -78,8 +78,9 @@ function EntityApi($http, $q) {
     function getEntityConfigInfo(applicationId, entityId) {
         return $http.get('/v1/applications/' + applicationId + '/entities/' + entityId + '/config', {observable: true, ignoreLoadingBar: true});
     }
-    function getEntityConfigState(applicationId, entityId) {
-        return $http.get('/v1/applications/' + applicationId + '/entities/' + entityId + '/config/current-state', {observable: true, ignoreLoadingBar: true});
+    function getEntityConfigState(applicationId, entityId, skipResolution) {
+        skipResolution = skipResolution ? true : false;
+        return $http.get('/v1/applications/' + applicationId + '/entities/' + entityId + '/config/current-state?suppressSecrets=true&skipReslution='+skipResolution, {observable: true, ignoreLoadingBar: true});
```

Comment:
I noticed a typo here, so I will merge this into my own branch first, to correct this and test everythingm and then submit everything as one PR.

Reply:
This is equivalent to:
```
function getEntityConfigState(applicationId, entityId, skipResolution) {
        return $http.get(`/v1/applications/${applicationId}/entities/${entityId}/config/current-state?suppressSecrets=true&skipReslution=${!!skipResolution}`, { observable: true, ignoreLoadingBar: true });
}
```

Reply:
it is in :)